amundsen_application/api/utils/search_utils.py

```diff
@@ -1,5 +1,10 @@
+import logging
+
 from typing import Dict, List  # noqa: F401
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
Do you intend to leave this here, or was this just for testing? Since these are utility methods that can be called by any APIs that need them, I'd expect any issues to be caught and logged from the API method as opposed to ever needing to log anything here.

Reply:
good catch, that was from local testing. i'll remove it. thanks!